test/utils/test_utils.py

```diff
@@ -12,14 +12,51 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
-import test.utils.local_mode as localmode
```

Comment:
Is local_mode.py still used? If not let's delete that.

Reply:
yeah, it should still be used by the other test files. I'll delete it with the last PR in this series of migrating the tests.